clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy

```diff
@@ -49,165 +31,53 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 class OperationsController {
 
-  @Autowired MessageSource messageSource
-  @Autowired OrchestrationProcessor orchestrationProcessor
-  @Autowired Registry registry
-  @Autowired (required = false) Collection<AllowedAccountsValidator> allowedAccountValidators = []
-  @Autowired (required = false) List<AtomicOperationDescriptionPreProcessor> atomicOperationDescriptionPreProcessors = []
-  @Autowired AtomicOperationsRegistry atomicOperationsRegistry
-  @Autowired SecurityConfig.OperationsSecurityConfigurationProperties opsSecurityConfigProps
-  @Autowired AccountCredentialsRepository accountCredentialsRepository
+  private final OperationsService operationsService
 
-  @Autowired DescriptionAuthorizer descriptionAuthorizer
-
-  /*
-   * APIs
-   * ----------------------------------------------------------------------------------------------------------------------------
-   */
-
-  /**
+  @Autowired
```

Comment:
`@Autowired` isn't required on ctors on Java classes, is it required for Groovy?